docs/concept/Generators.md

```diff
@@ -121,6 +121,22 @@ A few notes about sizes. iOS size has no upper limit, so be careful with the val
 
 Lastly, the splash-screen generator will exit if your input file has not been modified. The same source equality check, as the one on the app-icon generator, will encourage you to make customizations before using the generator (see the `--skip-source-equality-validation` section above).
 
+## CLI Options
+
+### `--exact`
+
+The default filename is PascalCased based on the name you pass in to the generate command. For example:
+
+`npx ignite-cli@latest g screen Episodes` will generate `EpisodesScreen.tsx` in the case of the default generator template `NAMEScreen.tsx`.
+
+This `--exact` switch specifies the generated filenames (`NAME` in the filename of your template) will be how you pass it in. For example:
+
+`npx ignite-cli@latest g screen log-in` will generate `log-inScreen.tsx`
+
+### `--destinationDir`
+
+Specifies the output path for the generated files. This will override the default path of `app/` (Ignite's path where all app code lives at the time of this writing) and any `destinationDir:` front matter that exists. This is useful in the case of file-based routing navigation systems, such as [Expo Router](https://docs.expo.dev/router/introduction/)
```

Comment:
```suggestion
Specifies the output path for the generated files. This will override the default path of `app/` (Ignite's path where all app code lives at the time of this writing) and any `destinationDir:` front matter that exists. This is useful in the case of file-based routing navigation systems, such as [Expo Router](https://docs.expo.dev/router/introduction/).
```